server/routes/adjustmentRoutes.ts

```diff
@@ -56,4 +57,20 @@ export default class AdjustmentRoutes {
     const { nomsId } = req.params
     return res.redirect(`${config.services.identifyRemandPeriods.url}/${nomsId}`)
   }
+
+  public additionalDays: RequestHandler = async (req, res): Promise<void> => {
+    const { caseloads, token } = res.locals.user
+    const { nomsId } = req.params
+    const prisonerDetail = await this.prisonerService.getPrisonerDetail(nomsId, caseloads, token)
+    const adjudicationsSearch = await this.prisonerService.getAdjudications(nomsId, token)
+    const adjudications = await Promise.all(
+      adjudicationsSearch.results.map(adj =>
+        this.prisonerService.getAdjudication(nomsId, adj.adjudicationNumber, token),
```

Comment:
@joelstobart-moj This is the only issue with this approach. There are two adjudications endpoints we're calling, the first is a search endpoint which returns all adjudications for an offender, the second endpoint we call for every adjudication, which could end in a lot of requests. We could add some filtering to the search endpoint so we only get the adjudications that resulted in ADAs.

Reply:
This would benefit from a [FUTURE] refactor to get all of the required data in one request, query, etc